### PR TITLE
linux/generic/hack-5.15: add missing patch header

### PR DESCRIPTION
**Pull request by feckert**
@Noltari @Ansuel This patch does not have a valid patch headers and does not apply on an external git tree with 'git am'. To fix this add the missing headers.

### CODE REVIEW

**Comment by Noltari** [comment]
Thanks @feckert!

**Comment by feckert** [comment]
@Noltari done

File: target/linux/generic/hack-5.15/810-bcma-ssb-fallback-sprom.patch [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+ drivers/bcma/sprom.c        | 23 ++++++++++++++---------
+ drivers/ssb/Kconfig         |  5 +++++
+ drivers/ssb/Makefile        |  1 +
+ drivers/ssb/main.c          |  8 ++++++++
+ drivers/ssb/sprom.c         | 12 +++++++++++-
+ drivers/ssb/ssb_private.h   |  4 ++++
+ 10 files changed, 60 insertions(+), 10 deletions(-)
+
+diff --git a/drivers/bcma/Kconfig b/drivers/bcma/Kconfig
+index 2e9732e5c565..eb02cf92a983 100644
```
Comment (Noltari) [inline target/linux/generic/hack-5.15/810-bcma-ssb-fallback-sprom.patch:21]: Did you refresh the kernel patches?
If you call `make target/linux/{clean,refresh} V=s` it should remove these lines (and for the other patched files as well)

Comment (feckert) [reply target/linux/generic/hack-5.15/810-bcma-ssb-fallback-sprom.patch:21]: No, I have not done that. But I will do it right away

Comment (feckert) [reply target/linux/generic/hack-5.15/810-bcma-ssb-fallback-sprom.patch]: done

**Review by Noltari — APPROVED** [review]
LGTM.

**Comment by Ansuel** [comment]
@feckert can you rebase this on top of master so i can flag this as merged?

**Pull request merged**